docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.9"
+services:
+  app:
+    restart: always
+    build:
+      dockerfile: Dockerfile
+      context: .
+    depends_on:
+      - db
+    volumes:
+    - .:/usr/src/app
+    
+  db:
+    image: postgres
+    ports:
+      - '4321:5432'
+    volumes:
+      - .data/postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: passwrd
+      POSTGRES_USER: root
+      POSTGRES_DB: discord_db
```

Comment:
As far as I understand, I have to run this command: `docker run --name postgres-docker -p 5432:5432 -e POSTGRES_PASSWORD=passwrd -e POSTGRES_USER=root -e POSTGRES_DB=discord_db -d postgres`
 to have a Postgres instance up and running and connect to it through another port (which is 4321 in this case).
Not sure if it's the right way, so will need your review on this

Reply:
That looks good to me. please make sure to add it to the `README`